### ISSUE TITLE
Translate local keyboard input to the target machine's layout

### ISSUE THREAD

**Issue by db39** [tiny-pilot/tinypilot#1386]
When your target machine uses a different keyboard layout to your client machine, it can be confusing to type a character only to end up with the incorrect input on the target machine.

Would it be possible to create a translation layer that allows users to specify the target machine's keyboard layout so that TinyPilot can map the keyboard input accordingly?

**Comment by mtlynch** [comment]
Yeah, this should be doable. We likely couldn't support every possible keyboard layout, but maybe we could support translations for common pairings.
